OpenRA.Mods.Cnc/Activities/LayMines.cs

```diff
@@ -137,7 +142,6 @@ void LayMine(Actor self)
 				var pool = ammoPools.FirstOrDefault(x => x.Info.Name == minelayer.Info.AmmoPoolName);
 				if (pool == null)
 					return;
-
```

Comment:
Any specific reason you removed this line? Imho having this new line is good for readability. (This is not really a blocker though.)